public/index.html

```diff
@@ -42,9 +42,6 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>SIE | Scientific Study Platform</title>
-    <!-- jsPsych library core script and styling -->
```

Comment:
Decided to remove JsPsych library & stylesheets since it is not needed throughout application.